### PR TITLE
Preparing Release 0.25

### PR DESCRIPTION
**Pull request by SergioEstevao**
PR to prepare the publishing the release 0.25

To test:
 - Check if README and CHANGELOG are correct.
 - Check if all correct changes are in.
 - Check if podspec is updated.

### CODE REVIEW

**Comment by SergioEstevao** [comment]
@frosty do you mind to give a general look here? I also have a test branch on the main app here: `issue/integrate_media_picker_0.25`

File: CHANGELOG.md [hunk 1]
```diff
@@ -14,6 +15,15 @@ All notable changes to this project will be documented in this file.
 - `0.16` Releases - [0.16](#16)
 - `0.15` Releases - [0.15](#15)
 
+---
+## [0.25](https://github.com/wordpress-mobile/MediaPicker-iOS/releases/tag/0.24)
```
Comment (frosty) [inline CHANGELOG.md]: Need to update the tags URL to 0.25.

File: CHANGELOG.md [hunk 2]
```diff
@@ -14,6 +15,15 @@ All notable changes to this project will be documented in this file.
...
+Released on 2017-11-17. All issues associated with this milestone can be found using this
+[filter](https://github.com/wordpress-mobile/MediaPicker-iOS/pulls?utf8=✓&q=is%3Apr%20is%3Aclosed%20milestone%3A0.25).
+
+### Added
+- Added the possibility to display a overlay view on top of the media cells. #259 #261
+- Added the possibility to search for assets on the picker. #257 #260 #268
```
Comment (frosty) [inline CHANGELOG.md]: The fix for when we reload cell content will also be in this update.

File: Pod/Classes/WPMediaPickerViewController.m [hunk 3]
```diff
@@ -453,7 +453,7 @@ -(void)updateDataWithRemoved:(NSIndexSet *)removed inserted:(NSIndexSet *)insert
         [self.collectionView performBatchUpdates:^{
             NSArray<NSIndexPath *> *indexPaths = [self indexPathsFromIndexSet:changed section:0];
             for (NSIndexPath *indexPath in indexPaths) {
-                WPMediaCollectionViewCell *cell = [self.collectionView cellForItemAtIndexPath:indexPath];
+                WPMediaCollectionViewCell *cell = (WPMediaCollectionViewCell *)[self.collectionView cellForItemAtIndexPath:indexPath];
```
Comment (frosty) [inline Pod/Classes/WPMediaPickerViewController.m]: whoops! Thanks for catching that.

**Comment by SergioEstevao** [comment]
@frosty Updating according to your suggestions.

**Pull request closed without merging**